lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -116,6 +151,12 @@ def unit_id(self) -> int:
         """ID of the current unit."""
         return int(self.unit.name.split("/")[1])
 
+    @property
+    def alternative_host(self) -> str:
+        """Return an alternative host (of another node) in case the current is offline."""
+        all_units_ips = units_ips(self, PEER)
+        return random.choice(list(all_units_ips.values()))
```

Comment:
Why doing this randomly instead of just picking the lower available unit?

Reply:
The reasoning behind was to sort of balance the load amongst all nodes, it probably doesn't matter much at this stage of the charm, though